### ISSUE TITLE
When user send answer on poll, telegram-bot don't know about poll_answer.

### ISSUE THREAD

**Issue by s-anches** [telegram-bot-rb/telegram-bot#132]
Hi. I need a make poll with bot. When user send answer on poll, telegram-bot call unsupported_payload_type method. TypedUpdate is included. This is normal?
```
class TelegramWebhooksController < Telegram::Bot::UpdatesController
  include Telegram::Bot::UpdatesController::MessageContext
  include Telegram::Bot::UpdatesController::CallbackQueryContext
  include Telegram::Bot::UpdatesController::Session
  include Telegram::Bot::UpdatesController::TypedUpdate

  self.session_store = :redis_store, {expires_in: 1.month}

  def message(message)
    true
  end

  def start!(*)
    if update.message.text === '/start'
      User.create_or_find_by(tg_id: update.message.from.id)
    end

    session.destroy

    buttons = {
      "📨 Пройти голосование": 'poll:start'
    }.map{|k, v| {text: k, callback_data: v}}

    respond_with :message, text: "Какой отчет вы хотите отправить?", reply_markup: {
      inline_keyboard: buttons.in_groups_of(2, false)
    }
  end

  def poll_callback_query(params = nil, *)
    save_context :poll_callback_query

    if params === 'start'
      poll = Poll.find(1)

      respond_with :poll,
      question: poll.question,
      options: poll.answers.map {|p| p.text},
      is_anonymous: false,
      type: 'regular',
      allows_multiple_answers: false
    end
  end

  def unsupported_payload_type
    if update.poll_answer
      true
    end
  end

end

**Comment by printercu** [comment]
Hi! What gem version do you use?

**Comment by printercu** [comment]
I see that update with these methods haven't been released, so it's available only in master. I'll release it later today.

**Comment by s-anches** [comment]
Now telegram-bot-ruby (0.12.0) and telegram-bot-types (0.6.2)

**Comment by printercu** [comment]
Just published 0.14.4, it provides support for polls.
